codegen/src/main/kotlin/org/web3j/openapi/codegen/servergen/ServerGenerator.kt

```diff
@@ -87,26 +91,24 @@ internal class ServerGenerator(
             StructExtensionsGenerator(
                 packageName = configuration.packageName,
                 contractName = it.contractDetails.contractName,
-                folderPath = Path.of(
-                    folderPath.substringBefore("kotlin"),
-                    "kotlin"
-                ).toString(),
+                folderPath = outputDir,
                 resourcesDefinition = it.contractDetails.abiDefinitions
             ).generate()
         }
     }
 
     private fun copyResources(folderPath: String) {
-        File(
-            Path.of(
-                folderPath.substringBefore("main"),
-                "main",
-                "resources"
-            ).toString()
-        ).apply {
-            mkdirs()
-        }
-        logger.debug("Copying server/resources")
+        // FIXME: Not needed if we won't copy the logging.properties
```

Comment:
this needs to sorted out before merge

Reply:
1. make a snapshot of web3j-openapi
2. use the snapshot in epirus-cli
3. run `installShadowDist` task in epirus-cli
4. generate a openpi project with your local epirus-cli which has the snapshot version of web3j-openapi and confirm that no logging errors are thrown

If so, remove this commented section

Reply:
keeping this comment here because it will be needed for the integration test later.